tests/test_core/test_alignment.py

```diff
@@ -2571,6 +2571,31 @@ def test_add_from_ref_aln(self):
             ValueError, aln1.add_from_ref_aln, aln2_wrong_refseq
         )  # test wrong_refseq
 
+    def test_annotate_matches_to(self):
+        """annotate_matches_to method should attach
+         annotations correctly to a Sequence object."""
+        seq = Sequence("TTCCACTTCCGCTT", name="x")
```

Comment:
logic of this test is incorrect. The Sequence classes are tested via the test_sequence.py test. This one should be a test of the Aligned class interface. Create this as follows
```python
aln = Alignment(dict(x="TTCCACTTCCGCTT"), moltype="dna")
seq = aln.named_seqs["x"]
```